src/main/java/ledger/user_interface/ui_controllers/component/AccountBalanceLabel.java

```diff
@@ -78,7 +78,7 @@ private void calculateBalanceForAccount() {
 
         String accountName = this.currentAccount.getName();
         if (accountName.length() > 20) accountName = accountName.substring(0, 17) + "...";
```

Comment:
It automatically truncates and adds ellipses to long names. If that's okay I can just remove that bit.